scipy/stats/_mannwhitneyu.py

```diff
@@ -157,27 +158,17 @@ def _base_case(m, n, k):
     return fmnks
 
 
-def _tie_term(ranks):
-    """Tie correction term"""
-    # element i of t is the number of elements sharing rank i
-    _, t = np.unique(ranks, return_counts=True, axis=-1)
-    return (t**3 - t).sum(axis=-1)
-
-
-def _get_mwu_z(U, n1, n2, ranks, axis=0, continuity=True):
+def _get_mwu_z(U, n1, n2, t, axis=0, continuity=True):
     '''Standardized MWU statistic'''
     # Follows mannwhitneyu [2]
     mu = n1 * n2 / 2
     n = n1 + n2
 
-    # Tie correction according to [2]
-    tie_term = np.apply_along_axis(_tie_term, -1, ranks)
+    # Tie correction according to [2], "Normal approximation and tie correction"
+    # "A more computationally-efficient form..."
```

Comment:
I think we should keep the comment “element i of tie_term is the number of elements sharing rank i” which was in `_tie_term` function to describe what is the `tie_term`.

Reply:
That described `t`, not the value returned by `_tie_term`. But I can describe what `t` is in `_rankata`, if you'd like.

Reply:
Oh, sorry. I missunderstood the code. 

> But I can describe what t is in _rankata, if you'd like. Please let me know.
Yes. I think describing what t is in _rankdata here or _randkada function is good idea.